react/features/base/react/components/web/Message.tsx

```diff
@@ -1,14 +1,21 @@
 import React, { Component, ReactNode } from 'react';
 import { toArray } from 'react-emoji-render';
+import { connect } from 'react-redux';
 
+import { IReduxState } from '../../../../app/types';
 import GifMessage from '../../../../chat/components/web/GifMessage';
 import { GIF_PREFIX } from '../../../../gifs/constants';
-import { isGifMessage } from '../../../../gifs/functions.web';
+import { isGifEnabled, isGifMessage } from '../../../../gifs/functions.web';
 
 import Linkify from './Linkify';
 
 interface IProps {
 
+    /**
+     * Whether the gifs are enabled or not.
+     */
+    gifEnabled?: boolean;
```

Comment:
Can you modify `isGifEnabled` to always return a boolean, it makes no sense that it can return undefined. This way it can default to false I guess, rather than haveing to do it everywhere.

Reply:
it actually always returns boolean.

Fixed!